### PR TITLE
make distclean in ci and fix bugs

### PR DESCRIPTION
**Pull request by sandisamp**
#### Describe Bug or Feature

* ci to take care of distclean if source is already configured
* updates to ci behaviour
* bug fixes

#### Describe Your Change

* Ci will exit if parameter key is provided wrong by the user, earlier it just gave a warning
* If provided container platform is not supported, throw warning 
* Display configuration params at the start of a ci run
* Add license,copyright and readme files to package
* Run PTL tests as pbsroot user instead of root
* Save PTL logs as 'logfile-{timestamp}' instead of the earlier 'logfile' so each instance of ptl run is not overwritten in logfile
* Run distclean if source is already configured. 

#### Attach Test and Valgrind Logs/Output

[build-server-centos-7-1.txt](https://github.com/openpbs/openpbs/files/5187621/build-server-centos-7-1.txt)
[logfile-2020-09-08-100813.txt](https://github.com/openpbs/openpbs/files/5187622/logfile-2020-09-08-100813.txt)

### CODE REVIEW

**Review by bhroam — APPROVED** [review]
LGTM

**Review by hirenvadalia — CHANGES_REQUESTED** [review]
Changes are fine, but holding merge as per request

**Review by hirenvadalia — APPROVED** [review]

**Review by hirenvadalia — APPROVED** [review]

File: ci/ci
Comment (bhroam) [inline ci/ci]: Did you mean '' without a space, or ' ' with a space?

Comment (hirenvadalia) [reply ci/ci]: @bhroam it doesn't matter actually, we just need to see that word (which is value of `node_val`) is in sys.argv or not.

**Review by bhroam — APPROVED** [review]

**Comment by bhroam** [comment]
I'm fine with the PR,  @hirenvadalia since you were holding off merging, I'll let you merge it when it is ready.

**Pull request merged**
